dashboard/lib/projects_list.rb

```diff
@@ -236,6 +247,7 @@ def fetch_featured_projects_by_type(project_type)
           project_type: project_type.to_s,
           state: 'active'
         ).
+        exclude(featured_at: nil).
```

Comment:
Why didn't we need this before?

Reply:
We didn't have 'bookmarked' featured projects before.

Reply:
status | featured_at | unfeatured_at
-- | -- | --
'bookmarked' | NULL | NULL
'active' | most recently featured at time stamp | NULL
'archived' | most recently featured at time stamp | most recently unfeatured at time stamp

Reply:
ah right...another instance where Rails would be a helpful (ie, it won't be super obvious to future readers that the combination of where(unfeatured_at: nil...) and exclude(featured_at: nil) is what gets you the right set of projects. Alas...

Reply:
I created a follow-up ticket to refactor `projects_list.rb` -> https://codedotorg.atlassian.net/browse/LABS-536
Thanks!